### ISSUE TITLE
TV pairings

### ISSUE THREAD

**Issue by skaincz** [inker/draw#56]
In the case of associations with two or more representatives, clubs have been paired in order to split their matches between Tuesday and Wednesday, namely:

Real Madrid & Barcelona
Atlético Madrid & Valencia
Bayern & Dortmund
Man. City & Tottenham
Juventus & Internazionale Milano
Paris & Lyon
Lokomotiv Moskva & CSKA Moskva
Porto & Benfica
Man. United & Liverpool
Napoli & Roma
Schalke & Hoffenheim
Ajax & PSV Eindhoven

UEFA.com

**Comment by skaincz** [comment]
As far as I know, it is NOT neccesary to split Teams from the same City to different days as it's always possible to give one team a home match and the other one an away match.

**Comment by Aule80** [comment]
You're right, UEFA doesn't even mention this (https://www.uefa.com/uefachampionsleague/news/newsid=2568286.html). I'll propose another change.

**Comment by Airmax77** [comment]
Sorry but i think thar're wrong this pair :
Man. City & Tottenham
Man. United & Liverpool
Juventus & Internazionale Milano
Napoli & Roma

I think  the pair implemented are :
Man. City & Man. United
Tottenham & Liverpool
Juventus & Napoli
Roma & Internazionale Milano

you can correct them?

**Comment by skaincz** [comment]
Look at this link ;-)

https://www.uefa.com/uefachampionsleague/news/newsid=2568286.html?iv=true

It really doesn't matter where the clubs are from. It's only about the popularity/ which Teams the spectators want to see. If there are teams from the same city, one of them plays at home and the other one away and  the other way.

**Comment by Airmax77** [comment]
I just read this page, and copy the point of pair club that are split : 
"In the case of associations with two or more representatives, clubs have been paired in order to split their matches between Tuesday and Wednesday, namely:

Real Madrid & Barcelona
Atlético Madrid & Valencia
Bayern & Dortmund
Man. City & Tottenham
Juventus & Internazionale Milano
Paris & Lyon
Lokomotiv Moskva & CSKA Moskva
Porto & Benfica
Man. United & Liverpool
Napoli & Roma
Schalke & Hoffenheim
Ajax & PSV Eindhoven

If a paired club is drawn, for example, in groups A, B, C or D, the other paired club – once drawn – will automatically be assigned to one groups E, F, G or H."

Home and away not influence the pair for the draw.
If Juventus (city Torino) is draw in group A, Internazionale (city Milano) cant draw in group B, C or D.

**Comment by TheAnswer1313** [comment]
You have to fix the pairings otherwise the tool isn't accurate.

**Comment by Airmax77** [comment]
I just read,.
I copy the point  : 
"In the case of associations with two or more representatives, clubs have been paired in order to split their matches between Tuesday and Wednesday, namely:

Real Madrid & Barcelona
Atlético Madrid & Valencia
Bayern & Dortmund
Man. City & Tottenham
Juventus & Internazionale Milano
Paris & Lyon
Lokomotiv Moskva & CSKA Moskva
Porto & Benfica
Man. United & Liverpool
Napoli & Roma
Schalke & Hoffenheim
Ajax & PSV Eindhoven

**If a paired club is drawn, for example, in groups A, B, C or D, the other paired club – once drawn – will automatically be assigned to one groups E, F, G or H.**"

You have correct all pair excluding England and Italy.

**Comment by skaincz** [comment]
Same procedure for Europa League:

https://www.uefa.com/uefaeuropaleague/news/newsid=2568277.html?iv=true

pairings:

Sevilla & Real Betis
Arsenal & Chelsea
Zenit & Spartak Moskva
Dynamo Kyiv & Vorskla
Beşiktaş & Fenerbahçe 
Salzburg & Rapid Wien
Olympiacos & PAOK
Anderlecht & Standard Liège 
Lazio & AC Milan
Marseille & Bordeaux
Celtic & Rangers 
RB Leipzig & Eintracht Frankfurt
Apollon Limassol & AEK Larnaca
Rosenborg & Sarpsborg
Slavia Praha & Jablonec

**Comment by Piggi95** [comment]
"The following pairings were announced by UEFA after the group stage teams were confirmed:
Juventus & Internazionale
Napoli & Atalanta"

**Comment by inker** [comment]
Saw that. Updated.

**Comment by lorenzocarli83** [comment]
I have compared the tv pairings in my database (derived from the official Uefa draw procedure, watched on youtube) with yours and found  a few mistakes/errors I would like to report. SO far i have only looked at seasons 09/10, 10/11 and 11/12

09/10: your pairing: Inter-Milan, Juve-Fiorentina. THis pairing can not be derived directly by watching the official Uefa draw so it can only be assumed. Based on my experience I would rather say that the pairings were instead Inter-Juve and MIlan-Fiorentina,

10/11: You paired Arsenal and Chelsea, just like in the season 09/10,but the actual paring for british teams was instead Chelsea-United , Arsenal-Tottenham, similar issue for the season 11/12 where the actual pairings were Chelsea-United and Arsenal-city.
In the same season Bayern was paired to Borussia D. rather than Leverkusen as estimated by you.

As soon as I will have the time I will also look at the other seasons.

**Comment by Piggi95** [comment]
AC Milan & Napoli
Juventus & Inter
